frontend/lib/repository/service/mbl.service.dart

```diff
@@ -53,7 +54,31 @@ class MblService {
     }
   }
 
-  dynamic getMeditations() {
-    // implement api call
+  Future<ApiResponse> getMeditations() async {
+    final response = await _httpClient.get(
+      getUrl(url: 'meditations'),
```

Comment:
please check the comment on the ticket again and add the populate parameter for audio and also the appropriate filters